mono/metadata/w32file-win32.c

```diff
@@ -515,6 +515,8 @@ mono_w32file_get_console_error (void)
 }
 #endif // HAVE_API_SUPPORT_WIN32_GET_STD_HANDLE
 
+#if HAVE_API_SUPPORT_WIN32_GET_FILE_SIZE_EX
```

Comment:
Can just GetFileSizeEx unconditionally, but ok either way.
i.e. GetFileSizeEx introduced in Windows 2000, no need to use GetFileSize. But it is ok.

Reply:
The previous one was just GetFileSize, so then it made sense, moving up to GetFileSizeEx drops the need to guard it as pointed out, but minor detail that I believe we can keep for now.